src/System.DirectoryServices.AccountManagement/src/System/DirectoryServices/AccountManagement/Utils.cs

```diff
@@ -45,12 +45,12 @@ internal static bool AreBytesEqual(byte[] src, byte[] tgt)
 
         internal static void ClearBit(ref int value, uint bitmask)
         {
-            value = (int)(((uint)value) & ((uint)(~bitmask)));
+            BitOps.ClearBit(ref value, (int)bitmask);
         }
 
         internal static void SetBit(ref int value, uint bitmask)
         {
-            value = (int)(((uint)value) | ((uint)bitmask));
+            BitOps.InsertBit(ref value, (uint)bitmask);
```

Comment:
I need to do some more analysis of the original code to make sure that the refactor is functionally equivalent.